reference_implementations/aws/online/terraform.tfvars

```diff
@@ -0,0 +1,24 @@
+project = "ai-deployment-bootcamp"
+region = "us-east-1"
+prefix = "poc"
+infra_env = "dev"
```

Comment:
Does this work similar to the [env var on the GCP implementation](https://github.com/VectorInstitute/ai-deployment-bootcamp/blob/main/reference_implementations/gcp/architectures/terraform.tfvars#L4) ([see also here](https://github.com/VectorInstitute/ai-deployment-bootcamp/blob/main/reference_implementations/gcp/architectures/online/main.tf#L164))? i.e. if I change the env to something like `env-marcelo` I can deploy a whole parallel architecture without affecting whatever has been deployed before?

Reply:
Yes, they're similar, with the difference that in AWS we don't have separate projects and we can use infra_env and append it to resource names or use it in other properties (e.g. Environment tags) to ensure isolation from other resources.

Reply:
Is this already being done, or do I need to manually append it in case I need to do that?

Reply:
No, I haven't done it so you need to append your env name to resources